docs/datascience/jupyter-notebooks.md

```diff
@@ -278,22 +278,38 @@ If you want to use the full set of debugging features supported in VS Code, such
 
 ![Debug cell button](images/jupyter/debug-cell.png)
 
+### Search through notebook
+
+You can search (Ctrl / Cmd + F) through an entire notebook or you can search parts of your notebook by filtering your search options. When searching through your notebook, click on the Filter option (funnel icon) to search across:
+- Markdown cell input (`Markdown Source`)
+- Markdown cell output (`Rendered Markdown`)
+- Code cell input (`Code Cell Source`)
+- Code cell output (`Cell Output`)
+
+Notebook searches are filtered cell inputs only by default.
+
+![Search options](images/jupyter/search-filter.png)
+
 ## Connect to a remote Jupyter server
 
 You can offload intensive computation in a Jupyter Notebook to other computers by connecting to a remote Jupyter server. Once connected, code cells run on the remote server rather than the local computer.
 
 To connect to a remote Jupyter server:
 
-1. Select the **Jupyter Server: local** button in the global Status bar or run the **Jupyter: Specify local or remote Jupyter server for connections** command from the Command Palette (`kb(workbench.action.showCommands)`).
+1. Open the Kernel Picker button on the top right-hand side of the notebook (or run the `Notebook: Select Notebook Kernel` command from the Command Palette).
```

Comment:
Please use bold to format commands and UI text. **Notebook: Select Notebook Kernal**